jazz-ui/src/environments/environment.oss.ts

```diff
@@ -1,17 +1,15 @@
 export const environment = {
   production:true,
   configFile: 'config/config.oss.json',
-  baseurl: "https://{API_GATEWAY_KEY_PROD}.execute-api.{inst_region}.amazonaws.com/prod",
+  baseurl: "https://v7m8a1nr30.execute-api.us-east-1.amazonaws.com/prod",
```

Comment:
why are these values being hardcoded now?

Reply:
this is a mistake from my part, let me quickly revert this back